### PR TITLE
feat: add autoware auto dependencies

### PR DESCRIPTION
**Pull request by 1222-takeshi**
## Related Issue(required)

#181 

## Description(required)
add autoware_auto dependencies for unnecessary packages.
## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: common/vehicle_constants_manager/package.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <buildtool_depend>autoware_auto_cmake</buildtool_depend>
+
+  <depend>autoware_auto_common</depend>
+  <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ament_lint_common</test_depend> -->
```
Comment (taikitanaka3) [inline common/vehicle_constants_manager/package.xml:18]:
```suggestion
  <test_depend>ament_lint_common</test_depend> 
```

Comment (1222-takeshi) [reply common/vehicle_constants_manager/package.xml:18]: I'm working on fixing the pre-commit now.
I'll fix it there at the same time. :bow:

Comment (1222-takeshi) [reply common/vehicle_constants_manager/package.xml]: fixed https://github.com/autowarefoundation/autoware.universe/pull/185/commits/50ab9b5e44fbe83fc3a4c898341dfd0a7a00fe27

**Comment by taikitanaka3** [comment]
@mitsudome-r @kenji-miyake 
should this pr be merged with tier4 format?  from  a05a6a4 ... 3d37a1c

this pr is related to this issue
https://github.com/autowarefoundation/autoware.universe/issues/117

**Comment by kenji-miyake** [comment]
@taikitanaka3 If we have to sync with the latest Autoware.Auto, no. Otherwise, I think yes.

**Review by taikitanaka3 — APPROVED** [review]

**Pull request merged**